web/templates/compose.hbs

```diff
@@ -26,9 +26,13 @@
             <span class="new_message_button mobile_button_container">
                 <button type="button" class="button small rounded compose_mobile_button"
                   id="left_bar_compose_mobile_button_big"
-                  title="{{t 'New message' }} (c)">
```

Comment:
Correction to the commit message: These are HTML title tooltips (https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title), not a Bootstrap thing.